docs/source/_static/version.js

```diff
@@ -48,7 +51,11 @@ window.addEventListener("DOMContentLoaded", () => {
             el.addEventListener("click", (e) => {
                 e.preventDefault();
                 try {
-                    const relativePart = window.location.pathname.replace(/^\/[^\/]+\//, "");
+                    let relativePart = window.location.pathname;
+                    // Remove the adbc/ prefix
+                    relativePart = relativePart.replace(/^\/adbc[^\/]+\//, "");
```

Comment:
Or we may want to use the following if `adbc*` part is optional.

```suggestion
                    let relativePart = window.location.pathname.replace(/^/\/, "");
                    // Remove the adbc/ prefix
                    relativePart = relativePart.replace(/^adbc[^\/]+\//, "");
```